themes/tiddlywiki/vanilla/base.tid

```diff
@@ -333,16 +333,17 @@ button svg, button img {
 }
 
 .tc-btn-boxed {
-	font-size: 0.6em;
-	padding: 0.2em;
+	padding: 0 0.1em 0.1em 0.1em;
 	margin: 1px;
+	line-height: 1em;
 	background: none;
 	border: 1px solid <<colour tiddler-controls-foreground>>;
 	border-radius: 0.25em;
 }
 
 html body.tc-body .tc-btn-boxed svg {
-	font-size: 1.6666em;
+	margin-top:-0.1em;
```

Comment:
hmm, why did you need such strange CSS rules. ... Are there some new DOM elements?